api/envoy/service/status/v3/csds.proto

```diff
@@ -64,6 +64,9 @@ message ClientStatusRequest {
 
   // The node making the csds request.
   config.core.v3.Node node = 2;
+  
+  // When true, response will include status, but no config.
```

Comment:
Can you elaborate on what is meant, i.e. exactly which fields are included. I vaguely remember discussions last synched timestamp etc.

Reply:
```suggestion
  // When true, response will include status, but Config Dump objects will contain only version_info field.
```

Reply:
Updated the description to specify which fields are and aren't populated.